src/libraries/System.Runtime.InteropServices.JavaScript/tests/System.Runtime.InteropServices.JavaScript.UnitTests/System/Runtime/InteropServices/JavaScript/HttpRequestMessageTest.cs

```diff
@@ -386,8 +386,7 @@ public void ToString_NonDefaultInstanceWithCustomHeaders_DumpAllFields(string ur
             Assert.Equal(
                 $"Method: PUT, RequestUri: '{uriData}', Version: 1.0, Content: " + typeof(StringContent).ToString() + ", Headers:" + Environment.NewLine +
                 "{" + Environment.NewLine +
-                "  Accept: text/plain; q=0.2" + Environment.NewLine +
-                "  Accept: text/xml; q=0.1" + Environment.NewLine +
+                "  Accept: text/plain; q=0.2, text/xml; q=0.1" + Environment.NewLine +
```

Comment:
cc @pavelsavara this is a test against `System.Net.Http` core functionality #42111, but landed it in S.R.InteropServices.Javascript for some reason. I assume the change doesn't break anything there, but please scream if yes.

Reply:
As long as the test passes, we are fine.
I guess the test is here because browser is messing with headers and you don't always get the same as in on wire.